lib/turbolinks.rb

```diff
@@ -30,16 +31,11 @@ def referer
 
         require 'action_dispatch/routing/redirection'
         ActionDispatch::Routing::Redirect.class_eval do
-          def call_with_turbolinks(env)
-            status, headers, body = call_without_turbolinks(env)
-
-            if env['rack.session'] && env['HTTP_X_XHR_REFERER']
-              env['rack.session'][:_turbolinks_redirect_to] = headers['Location']
-            end
-
-            [status, headers, body]
+          if defined?(prepend) && Rails.version >= '4'
```

Comment:
@kaspth without the check the tests don't pass on Rails 3.2 + Ruby 2.2. Something in 3.2 must be interfering with `prepend`…